src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
So something like this should do?
```
type RouterConfigurationResult = RouterConfiguration|any;
interface ConfigureRouter {
  configureRouter(config: RouterConfiguration, router: Router): RouterConfigurationResult|Promise<RouterConfigurationResult>|PromiseLike<RouterConfigurationResult>;
}
``` 
I may be overthinking it but it doesn't need to return exactly `RouterConfiguration`, it just checks if the returned value has an `exportToRouter(router: Router): void` and calls it. I probably should just leave it as is.